Libplanet.Extensions.Cocona/Commands/KeyCommand.cs

```diff
@@ -6,8 +6,10 @@ namespace Libplanet.Extensions.Cocona.Commands
     using System.Linq;
     using global::Cocona;
     using Libplanet.Crypto;
+    using Libplanet.Extensions.Cocona.Commands.Key;
     using Libplanet.KeyStore;
 
+    [HasSubCommands(typeof(DerivationCommand), "derive", Description = "Derive the address or public key from private key.")]
```

Comment:
Doesn't this project have lint for this?

```suggestion
    [HasSubCommands(
        typeof(DerivationCommand),
        "derive",
        Description = "Derive the address or public key from private key."
    )]
```

Reply:
No, it is there but it seems not worked. I'm checking why it doesn't work.

Reply:
I checked it. It was occurred because the `Libplanet.Extensions.Cocona` doesn't include `Menees.Analyzers.Settings.xml` and `stylecop.json` so the `Menees.Analyzers` in `Libplanet.Extensions.Cocona` uses 160 as `MaxLineColumns`. I will fix it in new pull request. Thanks 🙏🏻